### ISSUE TITLE
[EXPLORATION]: Disable known buggy axe rules individually

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#1140]
Migrated from the original ticket: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/8810

We should disable individual rules when overriding our axe configuration, rather than entire rulesets.

Relevant rules include: `aria-required-children`

This will involve changes to our configuration in general and updates to bring our app in line with any discovered failures once disabled rulesets are enabled.

Configuration changes include:

Updating individual tests:
```
-      client.axeCheck('.main', { rules: ['section508'] })
+      client.axeCheck('.main', { rules: { 'aria-required-children': { enabled: false } } })
```

`test/util/nightwatch-commands/axeCheck.js`
```
+  this.executeAsync((innerContext, options, done) => {
+    axe.run(document.querySelector(innerContext) || document, options, (err, results) => { // eslint-disable-line no-undef
        done({ err, results });
      });
 -  }, [context, (config || {}).rules || this.globals.rules || ['section508', 'wcag2a', 'wcag2aa']], response => {
 +  }, [context, Object.assign({}, this.globals, config)], response => {
```
`config/nightwatch.js`
```
     wcag2a: {
        globals: {
 -        rules: ['section508', 'wcag2a', 'wcag2aa']
 +        runOnly: {
 +          type: 'tag',
 +          values: ['section508', 'wcag2a', 'wcag2aa']
 +        }
        }
      }
```

**Comment by 1Copenut** [comment]
@meganhkelley I propose we close this ticket in favor of #16544 . I've got two SC in that issue to allow individual rules be toggled as part of that work.

**Comment by meganhkelley** [comment]
hey @1Copenut I'm not sure what team would own this now — possibly testing team? but I have no opposition to you closing axe tickets if it makes sense!

**Comment by 1Copenut** [comment]
@meganhkelley Yes, Testing Team will own it now. Mostly because I'm the one working on it, and I'm on the team. :). I'll go ahead and close this issue. Thank you!
